saas/fsm/suspicious_users/README.md

```diff
@@ -0,0 +1,28 @@
+# SaaS Manager - Suspicious Users
+
+## What it does
+
+This policy will raise an alert when Flexera SaaS Manager identifies suspicious users logging into SaaS applications.
```

Comment:
product doesn't want to use alert.  instead change **raise an alert** to **create an incident**.